.github/workflows/pull_request.yml

```
@@ -73,4 +73,5 @@ jobs:
     steps:
```

can you add following in this place, and try rerunning !
```suggestion
    permissions: write-all
    steps:
```